src/liquid/components/ld-button/ld-button.tsx

```diff
@@ -211,14 +211,14 @@ export class LdButton implements InnerFocusable {
         part="button focusable"
         ref={(el: HTMLAnchorElement | HTMLButtonElement) => (this.button = el)}
         rel={this.target === '_blank' ? 'noreferrer noopener' : undefined}
-        {...cloneAttributes(this.el, [
+        {...cloneAttributes.call(this, [
           'align-text',
           'justify-content',
           'mode',
           'progress',
           'size',
+          this.type === 'submit' ? 'type' : undefined, // submit is default
```

Comment:
how about enabling the same logic as in `getClassNames`?

```suggestion
          this.type === 'submit' && 'type', // submit is default
```

Reply:
Also, I think this might not even be necessary. Maybe we should actually think about removing `'submit'` as default value for this.type and keep in undefined by default...?

Reply:
Implicit form submission fails without the submit type being the default (tests fail). I'll keep it as is.